build.gradle

```diff
@@ -185,24 +258,24 @@ subprojects {
                     version project.version
                 }
             }
-        }.withXml {
-            project.configurations[JavaPlugin.RUNTIME_CONFIGURATION_NAME].allDependencies.findAll {
```

Comment:
Original purpose of this task was to generate a pom for local development use. 

Is the purpose of the change to this task to generate a pom for publishing?